beep/protocol/maccor_to_arbin.py

```diff
@@ -265,9 +297,9 @@ def compile_to_arbin(
             elif "Do" in step_abs["StepType"]:
                 loop_counter = int(re.search(r"\d+", step_abs["StepType"]).group())
                 blank_step["m_szStepCtrlType"] = "Set Variable(s)"
-                blank_step["m_szCtrlValue1"] = str(2 ** (loop_counter + 15))
-                blank_step["m_szExtCtrlValue1"] = "0"
+                blank_step["m_szCtrlValue"] = str(2 ** (loop_counter + 15))
```

Comment:
Are the hard-coded values for `blank_step["m_szCtrlValue"], blank_step["m_szCtrlValue1"]` project specific or project agnostic? Would it make sense to load them from a config file?

Reply:
These are control values that perform specific actions like "increment cycle index" or "reset loop counter" so they are project agnostic. Once we get them working correctly they should not need to be changed.

Reply:
These are control values that perform specific actions like "increment cycle index" or "reset loop counter" so they are project agnostic. Once we get them working correctly they should not need to be changed.